<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Karolina Smolinska-Garbulowska" />
  <title>Untitled</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="ht_9_12_files/reveal.js-3.3.0.1/css/reveal.css"/>



<link rel="stylesheet" href="ht_9_12_files/reveal.js-3.3.0.1/css/theme/simple.css" id="theme">


  <!-- some tweaks to reveal css -->
  <style type="text/css">
    .reveal h1 { font-size: 2.0em; }
    .reveal h2 { font-size: 1.5em;  }
    .reveal h3 { font-size: 1.25em;	}
    .reveal h4 { font-size: 1em;	}

    .reveal .slides>section,
    .reveal .slides>section>section {
      padding: 0px 0px;
    }



    .reveal table {
      border-width: 1px;
      border-spacing: 2px;
      border-style: dotted;
      border-color: gray;
      border-collapse: collapse;
      font-size: 0.7em;
    }

    .reveal table th {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      font-weight: bold;
      border-style: dotted;
      border-color: gray;
    }

    .reveal table td {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      border-style: dotted;
      border-color: gray;
    }


  </style>

    <style type="text/css">code{white-space: pre;}</style>

    <link rel="stylesheet" href="hygge.css"/>

<!-- Printing and PDF exports -->
<script id="paper-css" type="application/dynamic-css">

/* Default Print Stylesheet Template
   by Rob Glazebrook of CSSnewbie.com
   Last Updated: June 4, 2008

   Feel free (nay, compelled) to edit, append, and
   manipulate this file as you see fit. */


@media print {

	/* SECTION 1: Set default width, margin, float, and
	   background. This prevents elements from extending
	   beyond the edge of the printed page, and prevents
	   unnecessary background images from printing */
	html {
		background: #fff;
		width: auto;
		height: auto;
		overflow: visible;
	}
	body {
		background: #fff;
		font-size: 20pt;
		width: auto;
		height: auto;
		border: 0;
		margin: 0 5%;
		padding: 0;
		overflow: visible;
		float: none !important;
	}

	/* SECTION 2: Remove any elements not needed in print.
	   This would include navigation, ads, sidebars, etc. */
	.nestedarrow,
	.controls,
	.fork-reveal,
	.share-reveal,
	.state-background,
	.reveal .progress,
	.reveal .backgrounds {
		display: none !important;
	}

	/* SECTION 3: Set body font face, size, and color.
	   Consider using a serif font for readability. */
	body, p, td, li, div {
		font-size: 20pt!important;
		font-family: Georgia, "Times New Roman", Times, serif !important;
		color: #000;
	}

	/* SECTION 4: Set heading font face, sizes, and color.
	   Differentiate your headings from your body text.
	   Perhaps use a large sans-serif for distinction. */
	h1,h2,h3,h4,h5,h6 {
		color: #000!important;
		height: auto;
		line-height: normal;
		font-family: Georgia, "Times New Roman", Times, serif !important;
		text-shadow: 0 0 0 #000 !important;
		text-align: left;
		letter-spacing: normal;
	}
	/* Need to reduce the size of the fonts for printing */
	h1 { font-size: 28pt !important;  }
	h2 { font-size: 24pt !important; }
	h3 { font-size: 22pt !important; }
	h4 { font-size: 22pt !important; font-variant: small-caps; }
	h5 { font-size: 21pt !important; }
	h6 { font-size: 20pt !important; font-style: italic; }

	/* SECTION 5: Make hyperlinks more usable.
	   Ensure links are underlined, and consider appending
	   the URL to the end of the link for usability. */
	a:link,
	a:visited {
		color: #000 !important;
		font-weight: bold;
		text-decoration: underline;
	}
	/*
	.reveal a:link:after,
	.reveal a:visited:after {
		content: " (" attr(href) ") ";
		color: #222 !important;
		font-size: 90%;
	}
	*/


	/* SECTION 6: more reveal.js specific additions by @skypanther */
	ul, ol, div, p {
		visibility: visible;
		position: static;
		width: auto;
		height: auto;
		display: block;
		overflow: visible;
		margin: 0;
		text-align: left !important;
	}
	.reveal pre,
	.reveal table {
		margin-left: 0;
		margin-right: 0;
	}
	.reveal pre code {
		padding: 20px;
		border: 1px solid #ddd;
	}
	.reveal blockquote {
		margin: 20px 0;
	}
	.reveal .slides {
		position: static !important;
		width: auto !important;
		height: auto !important;

		left: 0 !important;
		top: 0 !important;
		margin-left: 0 !important;
		margin-top: 0 !important;
		padding: 0 !important;
		zoom: 1 !important;

		overflow: visible !important;
		display: block !important;

		text-align: left !important;
		-webkit-perspective: none;
		   -moz-perspective: none;
		    -ms-perspective: none;
		        perspective: none;

		-webkit-perspective-origin: 50% 50%;
		   -moz-perspective-origin: 50% 50%;
		    -ms-perspective-origin: 50% 50%;
		        perspective-origin: 50% 50%;
	}
	.reveal .slides section {
		visibility: visible !important;
		position: static !important;
		width: auto !important;
		height: auto !important;
		display: block !important;
		overflow: visible !important;

		left: 0 !important;
		top: 0 !important;
		margin-left: 0 !important;
		margin-top: 0 !important;
		padding: 60px 20px !important;
		z-index: auto !important;

		opacity: 1 !important;

		page-break-after: always !important;

		-webkit-transform-style: flat !important;
		   -moz-transform-style: flat !important;
		    -ms-transform-style: flat !important;
		        transform-style: flat !important;

		-webkit-transform: none !important;
		   -moz-transform: none !important;
		    -ms-transform: none !important;
		        transform: none !important;

		-webkit-transition: none !important;
		   -moz-transition: none !important;
		    -ms-transition: none !important;
		        transition: none !important;
	}
	.reveal .slides section.stack {
		padding: 0 !important;
	}
	.reveal section:last-of-type {
		page-break-after: avoid !important;
	}
	.reveal section .fragment {
		opacity: 1 !important;
		visibility: visible !important;

		-webkit-transform: none !important;
		   -moz-transform: none !important;
		    -ms-transform: none !important;
		        transform: none !important;
	}
	.reveal section img {
		display: block;
		margin: 15px 0px;
		background: rgba(255,255,255,1);
		border: 1px solid #666;
		box-shadow: none;
	}

	.reveal section small {
		font-size: 0.8em;
	}

}  
</script>


<script id="pdf-css" type="application/dynamic-css">
    
/**
 * This stylesheet is used to print reveal.js
 * presentations to PDF.
 *
 * https://github.com/hakimel/reveal.js#pdf-export
 */

* {
	-webkit-print-color-adjust: exact;
}

body {
	margin: 0 auto !important;
	border: 0;
	padding: 0;
	float: none !important;
	overflow: visible;
}

html {
	width: 100%;
	height: 100%;
	overflow: visible;
}

/* Remove any elements not needed in print. */
.nestedarrow,
.reveal .controls,
.reveal .progress,
.reveal .playback,
.reveal.overview,
.fork-reveal,
.share-reveal,
.state-background {
	display: none !important;
}

h1, h2, h3, h4, h5, h6 {
	text-shadow: 0 0 0 #000 !important;
}

.reveal pre code {
	overflow: hidden !important;
	font-family: Courier, 'Courier New', monospace !important;
}

ul, ol, div, p {
	visibility: visible;
	position: static;
	width: auto;
	height: auto;
	display: block;
	overflow: visible;
	margin: auto;
}
.reveal {
	width: auto !important;
	height: auto !important;
	overflow: hidden !important;
}
.reveal .slides {
	position: static;
	width: 100%;
	height: auto;

	left: auto;
	top: auto;
	margin: 0 !important;
	padding: 0 !important;

	overflow: visible;
	display: block;

	-webkit-perspective: none;
	   -moz-perspective: none;
	    -ms-perspective: none;
	        perspective: none;

	-webkit-perspective-origin: 50% 50%; /* there isn't a none/auto value but 50-50 is the default */
	   -moz-perspective-origin: 50% 50%;
	    -ms-perspective-origin: 50% 50%;
	        perspective-origin: 50% 50%;
}

.reveal .slides section {
	page-break-after: always !important;

	visibility: visible !important;
	position: relative !important;
	display: block !important;
	position: relative !important;

	margin: 0 !important;
	padding: 0 !important;
	box-sizing: border-box !important;
	min-height: 1px;

	opacity: 1 !important;

	-webkit-transform-style: flat !important;
	   -moz-transform-style: flat !important;
	    -ms-transform-style: flat !important;
	        transform-style: flat !important;

	-webkit-transform: none !important;
	   -moz-transform: none !important;
	    -ms-transform: none !important;
	        transform: none !important;
}

.reveal section.stack {
	margin: 0 !important;
	padding: 0 !important;
	page-break-after: avoid !important;
	height: auto !important;
	min-height: auto !important;
}

.reveal img {
	box-shadow: none;
}

.reveal .roll {
	overflow: visible;
	line-height: 1em;
}

/* Slide backgrounds are placed inside of their slide when exporting to PDF */
.reveal section .slide-background {
	display: block !important;
	position: absolute;
	top: 0;
	left: 0;
	width: 100%;
	z-index: -1;
}

/* All elements should be above the slide-background */
.reveal section>* {
	position: relative;
	z-index: 1;
}

/* Display slide speaker notes when 'showNotes' is enabled */
.reveal .speaker-notes-pdf {
	display: block;
	width: 100%;
	max-height: none;
	left: auto;
	top: auto;
	z-index: 100;
}

/* Display slide numbers when 'slideNumber' is enabled */
.reveal .slide-number-pdf {
	display: block;
	position: absolute;
	font-size: 14px;
}

</script>


<script>
var style = document.createElement( 'style' );
style.type = 'text/css';
var style_script_id = window.location.search.match( /print-pdf/gi ) ? 'pdf-css' : 'paper-css';
var style_script = document.getElementById(style_script_id).text;
style.innerHTML = style_script;
document.getElementsByTagName('head')[0].appendChild(style);
</script>

    <script src="ht_9_12_files/kePrint-0.0.1/kePrint.js"></script>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section>
    <h1 class="title">Untitled</h1>
  <h1 class="subtitle">Half-time presentation</h1>
    <h2 class="author">Karolina Smolinska-Garbulowska</h2>
    <h3 class="date">supervisor: Jan Komorowski <br><br> Upppsala, 2019-12-09</h3>
</section>

<section id="introduction" class="slide level2 emphasized" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Introduction</h2>
<p><span class="emphasized">Pay attention to this!</span></p>
<section style="text-align: left;">
Left-align: This works to align text left on this slide only.
</section>
</section>
<section id="section" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2></h2>
<p><img class="plain" src="images/study_schema.jpg" ></p>
</section>
<section id="project-1.-visunet-an-interactive-tool-for-network-visualization-of-rule-based-models" class="slide level2 emphasized" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 1. VisuNet: an interactive tool for network visualization of rule-based models</h2>
</section>
<section id="rule-network-rn-construction" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Rule-network (RN) construction</h2>
<p><img class="plain" src="images/schema.png"></p>
<p><img class="plain" src="images/Rule-network_contruction2.png"></p>
</section>
<section id="visunet" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>VisuNet</h2>
<p><img class="plain" src="images/VIsuNet_logo2copy.png" height="50"></p>
<ul>
<li>an R package</li>
<li>Publicly available on GitHub: <a href="https://github.com/komorowskilab/VisuNet" class="uri">https://github.com/komorowskilab/VisuNet</a></li>
</ul>
<p><img class="plain" src="images/interface.png" height="250"></p>
</section>
<section id="data-overview" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Data overview</h2>
<table class="table" style="font-size: 13px; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
Dataset
</th>
<th style="text-align:center;">
N.objects
</th>
<th style="text-align:center;">
Decision
</th>
<th style="text-align:center;">
Models
</th>
<th style="text-align:center;">
N.features
</th>
<th style="text-align:center;">
Model.quality
</th>
<th style="text-align:center;">
Visunet.Param
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
Autism (MG)
</td>
<td style="text-align:center;width: 10em; ">
146 patients: 82 autistic and 64 healthy boys
</td>
<td style="text-align:center;">
Autism vs Control
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
mcfs is running
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;vertical-align: top !important;" rowspan="4">
Amara (FB&amp;ZK)
</td>
<td style="text-align:center;width: 10em; vertical-align: top !important;" rowspan="4">
22 rhesus macaque: 10 infected in less than 4 challenges (Early) and 12 infected in more than 6 challenges(Late)
</td>
<td style="text-align:center;">
Early vs Late
</td>
<td style="text-align:center;">
W0
</td>
<td style="text-align:center;">
597
</td>
<td style="text-align:center;">
95%
</td>
<td style="text-align:center;">
Min. accuracy = 80%, min decision coverage =0.6, N top nodes=20
</td>
</tr>
<tr></li>
</ul></li>
</ul>
<td style="text-align:center;">
Early vs Late
</td>
<td style="text-align:center;">
W16
</td>
<td style="text-align:center;">
1068
</td>
<td style="text-align:center;">
100%
</td>
<td style="text-align:center;">
”
</td>
</tr>
<tr>
<td style="text-align:center;">
Early vs Late
</td>
<td style="text-align:center;">
W32
</td>
<td style="text-align:center;">
723
</td>
<td style="text-align:center;">
100%
</td>
<td style="text-align:center;">
”
</td>
</tr>
<tr>
<td style="text-align:center;">
Early vs Late
</td>
<td style="text-align:center;">
C1
</td>
<td style="text-align:center;">
448
</td>
<td style="text-align:center;">
95%
</td>
<td style="text-align:center;">
”
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
ULSAM (KD)
</td>
<td style="text-align:center;width: 10em; ">
963 patients: 134 T2D and 829 ND
</td>
<td style="text-align:center;">
ND (non-diabetes) vs T2D (type 2 diabetes)
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
67%
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;vertical-align: top !important;" rowspan="3">
Synthetic data (KS)
</td>
<td style="text-align:center;width: 10em; vertical-align: top !important;" rowspan="3">
120 obejcts: 60 D1 and 60 D2
</td>
<td style="text-align:center;">
D1 vs D2
</td>
<td style="text-align:center;">
weak correlation (0.2-0.5)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
59%
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr></li>
</ul></li>
</ul></li>
</ul>
<td style="text-align:center;">
D1 vs D2
</td>
<td style="text-align:center;">
medium correlation (0.2-0.7)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
76%
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
<tr></li>
</ul>
<td style="text-align:center;">
D1 vs D2
</td>
<td style="text-align:center;">
strong correlation (0.5 -0.9)
</td>
<td style="text-align:center;">
18
</td>
<td style="text-align:center;">
93%
</td>
<td style="text-align:center;">
<ul>
<li></td>
</tr>
</tbody>
</table></li>
</ul></li>
</ul>
<h5>
<p align="left">
MG - Mateusz Garbulowski
</p>
</h5>
<h5>
<p align="left">
FB&amp;ZK - Fredrik Barrenäs &amp; Zeeshan Khaliq
</p>
</h5>
<h5>
<p align="left">
KD - Klev Diamanti
</p>
</h5>
<h5>
<p align="left">
KS - Karolina Smolinska
</p>
</h5>
</section>
<section id="case-control-studies-of-autism-spectrum-disorders---sample-rn" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Case-control studies of autism spectrum disorders - sample RN</h2>
<p>Top 10% <iframe src='net.html' style='height:500px;width:700px'></iframe></p>
</section>
<section id="map7-and-cox2-interdependecies" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>MAP7 and COX2 interdependecies</h2>
</section>
<section id="project-1---summary" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 1 - Summary</h2>
</section>
<section id="project-2-funmotifs-and-the-functional-annotation-of-the-tissue-specific-noncoding-mutations" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 2: funMotifs and the functional annotation of the tissue-specific noncoding mutations</h2>
<p>Aims of the project were identification and analysis of significant functional motifs in non coding region of the human genome.</p>
<p>The</p>
<p>funMotifs was developed by Husen M. Umer and I was mainly working on the framework applications.</p>
<p>Two manuscripts were written based on the Project 2.</p>
</section>
<section id="funmotifs-freamwork" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>funMotifs freamwork</h2>
<div id="left">
<p><b> funMotifs content:</b></p>
<ul>
<li>Annotations for motifs of ~500 TFs in 15 tissues</li>
<li>Annotations:
<ul>
<li>TF peak,</li>
<li>DNase1 site,</li>
<li>Chromatin state,</li>
<li>Replication domain,</li>
<li>…</li>
</ul></li>
</ul>
<p><b> Definition of candidate functional regulatory mutations (CFRMs):</b></p>
<p>In the tissue(s) relevant for the variant.</p>
<ul>
<li>Overlap a TF motif and change entropy by at least |0.3|</li>
<li>TF expression &gt; 0</li>
<li>DNaseI peak</li>
<li>Functionality score &gt; 2.5 or TF-peak</li>
</ul>
<center>
<img class="plain" src="images/fun_score.png" height="200" >
</center>
</div>
<div id="right">
<p><img class="plain" src="images/funMotifs_schema.png" ></p>
</div>
</section>
<section id="manuscript-1" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Manuscript 1</h2>
<p><img class="plain" src="images/funMotifs_paper.png"  align="middle" style="border:1px solid black"></p>
<div id="left">
<ul>
<li>SNPs and short indels from 1000 Genome project (HU &amp; KS)</li>
<li>eQTL SNPs from the GTEx data portal v7 (HU &amp; KS)</li>
<li>SNPs from GWAS Catalogue and SNPs in LD with GWAS SNPs ()(KS)</li>
</ul>
<br><br>
<h5>
HS - Husen M. Umer
</h5>
<h5>
KS - Karolina Smolinska
</h5>
</div>
<div id="right">
<p><img class="plain" src="images/funmotifs_app.jpg" height="500" align="middle" ></p>
</div>
</section>
<section id="comparison-to-existing-annotation-databases" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Comparison to existing annotation databases</h2>
<p>2,219,330 eQTL SNPs were annotatied using funMotifs, RegulomeDB and HaploReg.</p>
<div id="left">
<table>
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:left;">
funMotifs
</th>
<th style="text-align:left;">
RegulomeDB
</th>
<th style="text-align:left;">
HaploReg
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Genome wide
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
Functionality score
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
</tr>
<tr>
<td style="text-align:left;">
Functional annotations
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
Tissue Specificity
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
No
</td>
</tr>
</tbody>
</table>
</div>
<div id="right">
<center>
<img class="plain" src="images/funmotif_comp.png" height="300">
</center>
</div>
<p><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br> RegulomeDB: Boyle,A.P., et al. (2012)</p>
<p>HaploReg: Ward,L.D., et al. (2016)</p>
</section>
<section id="overview-of-the-candidate-functional-gwas-snp-rs8103622-in-breast-tissue" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Overview of the candidate functional GWAS SNP rs8103622 in breast tissue</h2>
<center>
<img class="plain" src="images/funmotifs_snp.png" height="800">
</center>
</section>
<section id="pancancer" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Pancancer</h2>
<p><img class="plain" src="images/pancancer_schema.jpg" height="800" align="middle"></p>
</section>
<section id="project-2---summary" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 2 - Summary</h2>
</section>
<section id="future-plans" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Future plans</h2>
</section>
<section id="section-1" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2></h2>
<h5>
<p><strong>Publications: </strong></p>
<ol type="1">
<li><p>Husen M. Umer, <strong>Karolina Smolinska-Garbulowska</strong>, Nour-al-dain Marzouka, Zeeshan Khaliq, Claes Wadelius, Jan Komorowski, funMotifs: Tissue-specific transcription factor motifs, 2019, bioRxiv 683722; doi: <a href="https://doi.org/10.1101/683722" class="uri">https://doi.org/10.1101/683722</a> (subbmited)</p></li>
<li><p>Husen M. Umer, <strong>Karolina Smolinska-Garbulowska</strong>, Jan Komorowski, Claes Wadelius, Functional annotation of noncoding mutations in cancer, 2019(manuscript)</p></li>
<li><p>Marco Cavalli, Nicholas Baltzer, Gang Pan, José Ramón Bárcenas Walls, <strong>Karolina Smolinska Garbulowska</strong>, Chanchal Kumar, Stanko Skrtic, Jan Komorowski, and Claes Wadelius. Studies of liver tissue identifies functional gene regulatory elements associated to gene expression, type 2 diabetes and other metabolic diseases, Human genomics 13, no. 1 (2019): 20</p></li>
<li><p>Mateusz Garbulowski, Klev Diamanti, <strong>Karolina Smolinska-Garbulowska</strong>, Patricia Stoll, Susanne Bornelöv, Aleksander Øhrn, Jan Komorowski, (2019), R.ROSETTA: a package for analysis of rule-based classification models, bioRxiv 625905; doi: <a href="https://doi.org/10.1101/625905" class="uri">https://doi.org/10.1101/625905</a> These authors contributed equally to the work as second authors.</p>
</h5></li>
</ol>
</section>
<section id="section-2" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2></h2>
<h5>
<p><strong>Posters:</strong></p>
<ol type="1">
<li>Michal J. Dabrowski, Michal Draminski, Klev Diamanti, Mateusz Garbulowski, <strong>Karolina Smolinska</strong>, Jacek Koronacki, Jan Komorowski. Identification of the boundary between common and ancestry-dependent levels of gene expression of Homo sapiens adaptive immune system. Systems Biology: Networks, Cold Spring Harbor Laboratory, Cold Spring Harbor, USA 14-18 March 2017.</li>
<li>Mateusz Garbulowski, <strong>Karolina Smolinska</strong>, Klev Diamanti, Patricia Stoll, Zeeshan Khaliq, Sara Younes, Behrooz Torabi, Susanne Bornelöv, Jan Komorowski. RROSETTA: a toolkit for creating rough set models. Swedish Bioinformatics Workshop 2017, Uppsala, Sweden, 13-14 November 2017.</li>
<li>Mateusz Garbulowski, <strong>Karolina Smolinska</strong>, Klev Diamanti, Patricia Stoll, Zeeshan Khaliq, Sara Younes, Behrooz Torabi, Susanne Bornelöv, Jan Komorowski. RROSETTA: a toolkit for creating rough set models. Swedish Epigenetics and Chromatin Meeting, EpiChrom 2017, Uppsala, Sweden, 30 November - 1 December 2017.</li>
<li><strong>Karolina Smolinska-Garbulowska</strong>, Steven O. O. Anyango, Jan Komorowski, VisuNet: an interactive tool for rule network visualization of rule-based models, SciLifeLab Science Summit 2019, Artificial Intelligence for Life Sciences, Uppsala, May 15, 2019</li>
</ol>
<p><strong>Presentation: </strong></p>
<ol type="1">
<li><strong>Karolina Smolinska-Garbulowska</strong>, VisuNet: an interactive tool for rule network visualization of rule-based models, the awarded Flash Talk, SciLifeLab Science Summit 2019, Artificial Intelligence for Life Sciences, Uppsala, May 15, 2019</li>
</ol>
</h5>
</section>
<section id="acknowledgements" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Acknowledgements</h2>
<style type="text/css">
  .reveal p {
    text-align: left;
  }
  .reveal ul {
    display: block;
  }
  .reveal ol {
    display: block;
  }  
</style>
<p>Jan Komorowski Laboratory</p>
<p>Claes Wadelius Laboratory</p>
<p>Manfred Grabherr</p>
</section>
    </div>
  </div>

  <script src="ht_9_12_files/reveal.js-3.3.0.1/lib/js/head.min.js"></script>
  <script src="ht_9_12_files/reveal.js-3.3.0.1/js/reveal.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://github.com/hakimel/reveal.js#configuration
      Reveal.initialize({
        // Display controls in the bottom right corner
        controls: false,
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,
        // Enable keyboard shortcuts for navigation
        keyboard: true,
        // Vertical centering of slides
        center: false,
        // Enable slide navigation via mouse wheel
        mouseWheel: true,
        // Transition style
        transition: 'slide', // none/fade/slide/convex/concave/zoom
        // Transition style for full page slide backgrounds
        backgroundTransition: 'default', // none/fade/slide/convex/concave/zoom



        // Optional reveal.js plugins
        dependencies: [
        ]
      });
    </script>
  <!-- dynamically load mathjax for compatibility with self-contained -->
  <script>
    (function () {
      var script = document.createElement("script");
      script.type = "text/javascript";
      script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
      document.getElementsByTagName("head")[0].appendChild(script);
    })();
  </script>

<script>
  (function() {
    if (window.jQuery) {
      Reveal.addEventListener( 'slidechanged', function(event) {  
        window.jQuery(event.previousSlide).trigger('hidden');
        window.jQuery(event.currentSlide).trigger('shown');
      });
    }
  })();
</script>


  </body>
</html>
