<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Karolina Smolinska-Garbulowska" />
  <title>Computational methods for understanding complex human diseases</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="Presentation_ht_9_12_files/reveal.js-3.3.0.1/css/reveal.css"/>



<link rel="stylesheet" href="Presentation_ht_9_12_files/reveal.js-3.3.0.1/css/theme/simple.css" id="theme">


  <!-- some tweaks to reveal css -->
  <style type="text/css">
    .reveal h1 { font-size: 2.0em; }
    .reveal h2 { font-size: 1.5em;  }
    .reveal h3 { font-size: 1.25em;	}
    .reveal h4 { font-size: 1em;	}

    .reveal .slides>section,
    .reveal .slides>section>section {
      padding: 0px 0px;
    }



    .reveal table {
      border-width: 1px;
      border-spacing: 2px;
      border-style: dotted;
      border-color: gray;
      border-collapse: collapse;
      font-size: 0.7em;
    }

    .reveal table th {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      font-weight: bold;
      border-style: dotted;
      border-color: gray;
    }

    .reveal table td {
      border-width: 1px;
      padding-left: 10px;
      padding-right: 25px;
      border-style: dotted;
      border-color: gray;
    }


  </style>

    <style type="text/css">code{white-space: pre;}</style>

    <link rel="stylesheet" href="hygge.css"/>

<!-- Printing and PDF exports -->
<script id="paper-css" type="application/dynamic-css">

/* Default Print Stylesheet Template
   by Rob Glazebrook of CSSnewbie.com
   Last Updated: June 4, 2008

   Feel free (nay, compelled) to edit, append, and
   manipulate this file as you see fit. */


@media print {

	/* SECTION 1: Set default width, margin, float, and
	   background. This prevents elements from extending
	   beyond the edge of the printed page, and prevents
	   unnecessary background images from printing */
	html {
		background: #fff;
		width: auto;
		height: auto;
		overflow: visible;
	}
	body {
		background: #fff;
		font-size: 20pt;
		width: auto;
		height: auto;
		border: 0;
		margin: 0 5%;
		padding: 0;
		overflow: visible;
		float: none !important;
	}

	/* SECTION 2: Remove any elements not needed in print.
	   This would include navigation, ads, sidebars, etc. */
	.nestedarrow,
	.controls,
	.fork-reveal,
	.share-reveal,
	.state-background,
	.reveal .progress,
	.reveal .backgrounds {
		display: none !important;
	}

	/* SECTION 3: Set body font face, size, and color.
	   Consider using a serif font for readability. */
	body, p, td, li, div {
		font-size: 20pt!important;
		font-family: Georgia, "Times New Roman", Times, serif !important;
		color: #000;
	}

	/* SECTION 4: Set heading font face, sizes, and color.
	   Differentiate your headings from your body text.
	   Perhaps use a large sans-serif for distinction. */
	h1,h2,h3,h4,h5,h6 {
		color: #000!important;
		height: auto;
		line-height: normal;
		font-family: Georgia, "Times New Roman", Times, serif !important;
		text-shadow: 0 0 0 #000 !important;
		text-align: left;
		letter-spacing: normal;
	}
	/* Need to reduce the size of the fonts for printing */
	h1 { font-size: 28pt !important;  }
	h2 { font-size: 24pt !important; }
	h3 { font-size: 22pt !important; }
	h4 { font-size: 22pt !important; font-variant: small-caps; }
	h5 { font-size: 21pt !important; }
	h6 { font-size: 20pt !important; font-style: italic; }

	/* SECTION 5: Make hyperlinks more usable.
	   Ensure links are underlined, and consider appending
	   the URL to the end of the link for usability. */
	a:link,
	a:visited {
		color: #000 !important;
		font-weight: bold;
		text-decoration: underline;
	}
	/*
	.reveal a:link:after,
	.reveal a:visited:after {
		content: " (" attr(href) ") ";
		color: #222 !important;
		font-size: 90%;
	}
	*/


	/* SECTION 6: more reveal.js specific additions by @skypanther */
	ul, ol, div, p {
		visibility: visible;
		position: static;
		width: auto;
		height: auto;
		display: block;
		overflow: visible;
		margin: 0;
		text-align: left !important;
	}
	.reveal pre,
	.reveal table {
		margin-left: 0;
		margin-right: 0;
	}
	.reveal pre code {
		padding: 20px;
		border: 1px solid #ddd;
	}
	.reveal blockquote {
		margin: 20px 0;
	}
	.reveal .slides {
		position: static !important;
		width: auto !important;
		height: auto !important;

		left: 0 !important;
		top: 0 !important;
		margin-left: 0 !important;
		margin-top: 0 !important;
		padding: 0 !important;
		zoom: 1 !important;

		overflow: visible !important;
		display: block !important;

		text-align: left !important;
		-webkit-perspective: none;
		   -moz-perspective: none;
		    -ms-perspective: none;
		        perspective: none;

		-webkit-perspective-origin: 50% 50%;
		   -moz-perspective-origin: 50% 50%;
		    -ms-perspective-origin: 50% 50%;
		        perspective-origin: 50% 50%;
	}
	.reveal .slides section {
		visibility: visible !important;
		position: static !important;
		width: auto !important;
		height: auto !important;
		display: block !important;
		overflow: visible !important;

		left: 0 !important;
		top: 0 !important;
		margin-left: 0 !important;
		margin-top: 0 !important;
		padding: 60px 20px !important;
		z-index: auto !important;

		opacity: 1 !important;

		page-break-after: always !important;

		-webkit-transform-style: flat !important;
		   -moz-transform-style: flat !important;
		    -ms-transform-style: flat !important;
		        transform-style: flat !important;

		-webkit-transform: none !important;
		   -moz-transform: none !important;
		    -ms-transform: none !important;
		        transform: none !important;

		-webkit-transition: none !important;
		   -moz-transition: none !important;
		    -ms-transition: none !important;
		        transition: none !important;
	}
	.reveal .slides section.stack {
		padding: 0 !important;
	}
	.reveal section:last-of-type {
		page-break-after: avoid !important;
	}
	.reveal section .fragment {
		opacity: 1 !important;
		visibility: visible !important;

		-webkit-transform: none !important;
		   -moz-transform: none !important;
		    -ms-transform: none !important;
		        transform: none !important;
	}
	.reveal section img {
		display: block;
		margin: 15px 0px;
		background: rgba(255,255,255,1);
		border: 1px solid #666;
		box-shadow: none;
	}

	.reveal section small {
		font-size: 0.8em;
	}

}  
</script>


<script id="pdf-css" type="application/dynamic-css">
    
/**
 * This stylesheet is used to print reveal.js
 * presentations to PDF.
 *
 * https://github.com/hakimel/reveal.js#pdf-export
 */

* {
	-webkit-print-color-adjust: exact;
}

body {
	margin: 0 auto !important;
	border: 0;
	padding: 0;
	float: none !important;
	overflow: visible;
}

html {
	width: 100%;
	height: 100%;
	overflow: visible;
}

/* Remove any elements not needed in print. */
.nestedarrow,
.reveal .controls,
.reveal .progress,
.reveal .playback,
.reveal.overview,
.fork-reveal,
.share-reveal,
.state-background {
	display: none !important;
}

h1, h2, h3, h4, h5, h6 {
	text-shadow: 0 0 0 #000 !important;
}

.reveal pre code {
	overflow: hidden !important;
	font-family: Courier, 'Courier New', monospace !important;
}

ul, ol, div, p {
	visibility: visible;
	position: static;
	width: auto;
	height: auto;
	display: block;
	overflow: visible;
	margin: auto;
}
.reveal {
	width: auto !important;
	height: auto !important;
	overflow: hidden !important;
}
.reveal .slides {
	position: static;
	width: 100%;
	height: auto;

	left: auto;
	top: auto;
	margin: 0 !important;
	padding: 0 !important;

	overflow: visible;
	display: block;

	-webkit-perspective: none;
	   -moz-perspective: none;
	    -ms-perspective: none;
	        perspective: none;

	-webkit-perspective-origin: 50% 50%; /* there isn't a none/auto value but 50-50 is the default */
	   -moz-perspective-origin: 50% 50%;
	    -ms-perspective-origin: 50% 50%;
	        perspective-origin: 50% 50%;
}

.reveal .slides section {
	page-break-after: always !important;

	visibility: visible !important;
	position: relative !important;
	display: block !important;
	position: relative !important;

	margin: 0 !important;
	padding: 0 !important;
	box-sizing: border-box !important;
	min-height: 1px;

	opacity: 1 !important;

	-webkit-transform-style: flat !important;
	   -moz-transform-style: flat !important;
	    -ms-transform-style: flat !important;
	        transform-style: flat !important;

	-webkit-transform: none !important;
	   -moz-transform: none !important;
	    -ms-transform: none !important;
	        transform: none !important;
}

.reveal section.stack {
	margin: 0 !important;
	padding: 0 !important;
	page-break-after: avoid !important;
	height: auto !important;
	min-height: auto !important;
}

.reveal img {
	box-shadow: none;
}

.reveal .roll {
	overflow: visible;
	line-height: 1em;
}

/* Slide backgrounds are placed inside of their slide when exporting to PDF */
.reveal section .slide-background {
	display: block !important;
	position: absolute;
	top: 0;
	left: 0;
	width: 100%;
	z-index: -1;
}

/* All elements should be above the slide-background */
.reveal section>* {
	position: relative;
	z-index: 1;
}

/* Display slide speaker notes when 'showNotes' is enabled */
.reveal .speaker-notes-pdf {
	display: block;
	width: 100%;
	max-height: none;
	left: auto;
	top: auto;
	z-index: 100;
}

/* Display slide numbers when 'slideNumber' is enabled */
.reveal .slide-number-pdf {
	display: block;
	position: absolute;
	font-size: 14px;
}

</script>


<script>
var style = document.createElement( 'style' );
style.type = 'text/css';
var style_script_id = window.location.search.match( /print-pdf/gi ) ? 'pdf-css' : 'paper-css';
var style_script = document.getElementById(style_script_id).text;
style.innerHTML = style_script;
document.getElementsByTagName('head')[0].appendChild(style);
</script>

    <script src="Presentation_ht_9_12_files/kePrint-0.0.1/kePrint.js"></script>
</head>
<body>
  <div class="reveal">
    <div class="slides">

<section>
    <h1 class="title">Computational methods for understanding complex human diseases</h1>
  <h1 class="subtitle">Half-time presentation</h1>
    <h2 class="author">Karolina Smolinska-Garbulowska</h2>
    <h3 class="date">supervisor: Jan Komorowski <br><br> Upppsala, 2019-12-09</h3>
</section>

<section id="introduction" class="slide level2 emphasized" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Introduction</h2>
<p>Uncovering explainability for the complex health-related problems is a crucial task.</p>
<p>Large number of available data complicates the process of discovery the most informative features for the biological outcomes.</p>
<p>We focused on developing methodologies that support the disease discovery process and as a result two tools were implemented.</p>
<p>The main aim of my projects was to improve the methods and apply them to large-scale experiments performed on complex diseases, e.g. cancer and autism.</p>
<p><br><br><br><br> <img class="plain" src="images/study_schema.png" ></p>
</section>
<section id="project-1.-visunet-an-interactive-tool-for-network-visualization-of-rule-based-models" class="slide level2 emphasized" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 1. VisuNet: an interactive tool for network visualization of rule-based models</h2>
<p>Explainability make classifiers biologically meaningful</p>
<p><img class="plain" src="images/schema.png"></p>
<p>Explainability:</p>
<ul>
<li>What interdependencies of genes/variables contribute to classification?</li>
<li>What variables/genes are more important?</li>
</ul>
</section>
<section id="rule-network-rn-construction" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Rule-network (RN) construction</h2>
<p>Interdependencies are discovered from transparent classifiers, such as rule-based classifiers (RBCs).</p>
<p><img class="plain" src="images/Rule-network_contruction2.png"></p>
<p><br><br> The conjuncts are co-predictive of the decision.</p>
<p>Nodes - conjuncts in rules, i.e. features with their values occurring in the rules.</p>
<p>Edge - connects two nodes if the corresponding conjuncts occur in the same rule.</p>
<p>RNs are built for each decision (outcome class).</p>
</section>
<section id="visunet" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>VisuNet</h2>
<center>
<img class="plain" src="images/VIsuNet_logo2copy.png" height="100">
</center>
<ul>
<li>RN visualization tool implemented as an R package,</li>
<li>helps identifying the strongest features and their interdependencies,</li>
<li>easy to use due to an interactive interface,</li>
<li>applicable for any RBC,</li>
<li>publicly available on GitHub: <a href="https://github.com/komorowskilab/VisuNet" class="uri">https://github.com/komorowskilab/VisuNet</a></li>
</ul>
<center>
<img class="plain" src="images/interface.png" height="250">
</center>
</section>
<section id="applications" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Applications</h2>
<table class="table" style="font-size: 13px; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
Dataset
</th>
<th style="text-align:center;">
Data type
</th>
<th style="text-align:center;">
#objects
</th>
<th style="text-align:center;">
Decision
</th>
<th style="text-align:center;">
Models
</th>
<th style="text-align:center;">
#features
</th>
<th style="text-align:center;">
Rule-based model quality
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
Autism (MG)<sup>1</sup>
</td>
<td style="text-align:center;width: 10em; vertical-align: middle !important;" rowspan="5">
gene expression levels
</td>
<td style="text-align:center;">
146 patients: 82 autistic and 64 healthy boys
</td>
<td style="text-align:center;">
Autism vs Control
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
35
</td>
<td style="text-align:center;">
80%
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;vertical-align: middle !important;" rowspan="4">
SIV (FB&amp;ZK)<sup>2</sup>
</td>
<td style="text-align:center;vertical-align: middle !important;" rowspan="4">
22 Rhesus macaques: 10 infected in less than 4 challenges (Early) and 12 infected in more than 6 challenges (Late) with SIV after vaccination
</td>
<td style="text-align:center;vertical-align: middle !important;" rowspan="4">
Early vs Late
</td>
<td style="text-align:center;">
W0
</td>
<td style="text-align:center;">
597
</td>
<td style="text-align:center;">
95%
</td>
</tr>
<tr>
<td style="text-align:center;">
W16
</td>
<td style="text-align:center;">
1068
</td>
<td style="text-align:center;">
100%
</td>
</tr>
<tr>
<td style="text-align:center;">
W32
</td>
<td style="text-align:center;">
723
</td>
<td style="text-align:center;">
100%
</td>
</tr>
<tr>
<td style="text-align:center;">
C1
</td>
<td style="text-align:center;">
448
</td>
<td style="text-align:center;">
95%
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
ULSAM (KD)<sup>3</sup>
</td>
<td style="text-align:center;width: 10em; ">
metabolite levels
</td>
<td style="text-align:center;">
963 patients: 134 type 2 diabetes (T2D) and 829 non-diabetes (ND)
</td>
<td style="text-align:center;">
ND vs T2D
</td>
<td style="text-align:center;">
<ul>
<li></td>
<td style="text-align:center;">
19
</td>
<td style="text-align:center;">
67%
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
Synthetic data (KS)<sup>4</sup>
</td>
<td style="text-align:center;width: 10em; ">
synthetically correlated variables
</td>
<td style="text-align:center;">
500 objects: 250 D1 and 250 D2
</td>
<td style="text-align:center;">
D1 vs D2
</td>
<td style="text-align:center;">
100 models
</td>
<td style="text-align:center;">
25
</td>
<td style="text-align:center;">
48% - 100%
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<span style="font-style: italic;">Note: </span>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup></sup> Data sorce
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup>1</sup> Alter, M.D., et al., 2011
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup>2</sup> Barrenäs, F., et al., 2019
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup>3</sup>
</td>
</tr>
<tr>
<td style="padding: 0; border: 0;" colspan="100%">
<sup>4</sup> Garbulowski, M., et al., 2019
</td>
</tr>
</tfoot>
</table></li>
</ul></li>
</ul>
<br><br>
<h5>
<p align="left">
MG - Mateusz Garbulowski
</p>
</h5>
<h5>
<p align="left">
FB&amp;ZK - Fredrik Barrenäs &amp; Zeeshan Khaliq
</p>
</h5>
<h5>
<p align="left">
KD - Klev Diamanti
</p>
</h5>
<h5>
<p align="left">
KS - Karolina Smolinska
</p>
</h5>
</section>
<section id="case-control-studies-of-autism---sample-rn" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Case-control studies of autism - sample RN</h2>
<iframe src="images/net_Autism_fig_new1.html" style="height:800px;width:800px">
</iframe>
</section>
<section id="case-control-studies-of-autism---sample-rn-1" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Case-control studies of autism - sample RN</h2>
<center>
<img class="plain" src="images/Autism_net_cust.png" height="500">
</center>
<p>4 genes were previously reported in the SFARI Gene database of autism associations: COX2, TMLHE, NCKAP5L and TSPOAP1.</p>
<p>Several genes identified in the literature as autism associated: ZSCAN18, SCIN, NCS1, CAPS2, ZFP36L2.</p>
</section>
<section id="map7-and-cox2-interdependency-in-autism" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>MAP7 and COX2 interdependency in autism</h2>
<div id="left">
<p>The strongest interdependency is visible between COX2 and MAP7.</p>
<p>COX2 regulates anxiety behaviour in the mouse model (Wong, C.T., et al., 2019)</p>
<p>MAP7 is required for axon growth and branch formation during embryonic brain formation. The association of MAP7 with autism and schizophrenia was previously reported (Tymanskyj, S.R., et al., 2017)</p>
<p>RN may suggest a link between embryonic development events and anxiety behaviour.</p>
</div>
<div id="right">
<center>
<img class="plain" src="images/COX2.png" align="middle">
</center>
</div>
</section>
<section id="project-1---summary" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 1 - Summary</h2>
<ul>
<li>VisuNet is conceptually simple, but useful for interpreting RBCs,</li>
<li>RN identifies different combinations of features and their levels that are co-predictors,</li>
<li>RN helps explain the differences between the decision classes,</li>
<li>autism-control RN confirms genes associations with autism,</li>
<li>RN may help to discover the interaction mechanisms for attributes, e.g. MAP7 and CAPS2.</li>
</ul>
<p>The manuscript is planned to finish by the end of this year.</p>
</section>
<section id="project-2-funmotifs-and-the-functional-annotation-of-the-tissue-specific-noncoding-mutations" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 2: funMotifs and the functional annotation of the tissue-specific noncoding mutations</h2>
<p>The gene expression is regulated by transcription factors (TFs) through binding to TF biding sites (motifs).<br />
To understand how the genes are regulated, we need to know what determinated functionality of TF motifs.</p>
<p>We incorporated data from large-scale genomics platforms including ENCODE, RoadMap Epigenomics and FANTOM to characterize the tissue-specific functional TF motifs.</p>
<p>As a consequence, funMotifs was developed by Husen M. Umer. I was mainly working on the framework applications.</p>
<p>Two manuscripts were written based on the Project 2.</p>
</section>
<section id="manuscript-1" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Manuscript 1</h2>
<br><br><br>
<center>
<img class="plain" src="images/funMotifs_paper.png"  align="middle" style="border:1px solid black">
<center>
</section>
<section id="funmotifs-framework" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>funMotifs framework</h2>
<div id="left">
<p><b> funMotifs content:</b></p>
<ul>
<li>Annotations for motifs of ~500 TFs in 15 tissues</li>
<li>Annotations:
<ul>
<li>TF peak,</li>
<li>DNase1 site,</li>
<li>Chromatin state,</li>
<li>Replication domain,</li>
<li>…</li>
</ul></li>
</ul>
<p><b> Definition of candidate functional regulatory mutations (CFRMs):</b></p>
<p>In the tissue(s) relevant for the variant.</p>
<ul>
<li>Overlap a TF motif and change entropy by at least |0.3|,</li>
<li>TF expression &gt; 0,</li>
<li>DNaseI peak,</li>
<li>Functionality score &gt; 2.5 or TF-peak.</li>
</ul>
</div>
<div id="right">
<p><img class="plain" src="images/funMotifs_schema.png" ></p>
</div>
</section>
<section id="cfrms-distribution" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>CFRMs distribution</h2>
<center>
<img class="plain" src="images/mut_ov.png" style="max-width:105%;">
<center>
</section>
<section id="applications-1" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Applications</h2>
<div id="left">
<ul>
<li>SNPs and short indels from 1000 Genome project (HU &amp; KS)</li>
<li>eQTL SNPs from the GTEx data portal v7 (HU &amp; KS)</li>
<li>SNPs from GWAS Catalogue and SNPs in LD with GWAS SNPs for r&gt;0.8 (KS)</li>
</ul>
</div>
<div id="right">
<p><img class="plain" src="images/funmotifs_app.png" height="450" align="middle" ></p>
</div>
<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
<h5>
<p align="left">
HS - Husen M. Umer
</p>
</h5>
<h5>
<p align="left">
KS - Karolina Smolinska
</p>
</h5>
</section>
<section id="comparison-to-existing-annotation-databases" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Comparison to existing annotation databases</h2>
<p>2,219,330 eQTL SNPs were annotated using funMotifs, RegulomeDB and HaploReg.</p>
<div id="left">
<table class="table" style="font-size: 18px; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:left;">
funMotifs
</th>
<th style="text-align:left;">
RegulomeDB
</th>
<th style="text-align:left;">
HaploReg
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Genome wide
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
Functionality score
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
</tr>
<tr>
<td style="text-align:left;">
Functional annotations
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
Yes
</td>
</tr>
<tr>
<td style="text-align:left;">
Tissue Specificity
</td>
<td style="text-align:left;">
Yes
</td>
<td style="text-align:left;">
No
</td>
<td style="text-align:left;">
No
</td>
</tr>
</tbody>
</table>
</div>
<div id="right">
<center>
<img class="plain" src="images/funmotif_comp.png" height="450">
</center>
</div>
<p><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br> RegulomeDB: Boyle,A.P., et al. (2012)</p>
<p>HaploReg: Ward,L.D., et al. (2016)</p>
</section>
<section id="overview-of-the-candidate-functional-gwas-snp-rs8103622-in-breast-tissue" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Overview of the candidate functional GWAS SNP rs8103622 in breast tissue</h2>
<center>
<img class="plain" src="images/funmotifs_snp.png" height="600">
</center>
</section>
<section id="manuscript-2" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Manuscript 2</h2>
<center>
<img class="plain" src="images/MS_2.png" align="middle" style="border:1px solid black">
</center>
<p><b>The aim of the study</b></p>
<p>Functional annotation of mutations in the Pan Cancer Analysis of Whole Genomes consortium data for 2,500 cancer genomes in 37 cancer types.</p>
<p><b>Results</b></p>
<p>Using the funMotifs framework we identified 121,029 CFRMs overlapped 217,075 potentially functional motifs across the 44 cohorts. Many CFRMs were found in genes that were previously linked with cancer.</p>
</section>
<section id="overview-of-mutations" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Overview of mutations</h2>
<center>
<img class="plain" src="images/data_overwiev.png" >
</center>
</section>
<section id="workflow" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Workflow</h2>
<center>
<img class="plain" src="images/pancancer_schema.png" height="650" align="middle">
</center>
</section>
<section id="cebpb---the-most-significantly-mutated-motifs" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>CEBPB - the most significantly mutated motifs</h2>
<center>
<img class="plain" src="images/cebp.png" height="500">
</center>
<p>The active motifs of CEBPB are significantly less methylated genome-wide than inactive motifs and still there is an abundance of mutations at this site (P-value &lt; 1e-50).</p>
</section>
<section id="highly-recurrent-mutated-regulatory-elements" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Highly recurrent mutated regulatory elements</h2>
<center>
<img class="plain" src="images/Myc.png" height="500">
</center>
</section>
<section id="project-2---summary" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Project 2 - Summary</h2>
<ul>
<li>Characterization of the noncoding regions of human genome is important to understand mechanisms that lead to complex diseases,</li>
<li>Functional annotations help to identify regulatory mutations that may significantly dysregulate gene expression in cancer and other complex diseases,</li>
<li>funMotifs applications identified potential functional variants in cancer and normal genomes,</li>
<li>Experimental validation may be performed on the CFRMs to confirm our findings</li>
</ul>
<p>funMotifs was implemented as a command-line pipeline (<a href="https://github.com/komorowskilab/funMotifs" class="uri">https://github.com/komorowskilab/funMotifs</a>) and a web-based interface: <a href="http://bioinf.icm.uu.se/funmotifs" class="uri">http://bioinf.icm.uu.se/funmotifs</a>.</p>
</section>
<section id="future-plans" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Future plans</h2>
<ul>
<li><p>Project 1. - implement the possibility to annotate the RN attributes using biological properties of the input datatypes and various databases, e.g. GO terms for a gene expression data.</p></li>
<li><p>Project 2. - modification of the funMotifs algorithm to improve identification of the false positive regulatory motifs. We are planning to continue research on the functional annotation of motifs. We would like to extend the functional motifs definition in terms of the conservation of the individual nucleotides in TF motifs across various diseases.</p></li>
<li><p>Project 3. - identification and visualization of interdependent genes that discern patients in the case-control studies of autism spectrum disorders by combining multiple rule-based classifiers.</p></li>
<li><p>Project 4. - identification of potential biomarkers that discern glioma subtypes using CFRMs from the COSMIC-available mutations for gliomas. Intersection of CFRMs with the differentially expressed genes based on the TCGA and GTEx databases.</p></li>
</ul>
</section>
<section id="section" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2></h2>
<h5>
<p><strong>Publications: </strong></p>
<ol type="1">
<li><p>Husen M. Umer, <strong>Karolina Smolinska-Garbulowska</strong>, Nour-al-dain Marzouka, Zeeshan Khaliq, Claes Wadelius, Jan Komorowski, funMotifs: Tissue-specific transcription factor motifs, 2019, bioRxiv 683722; doi: <a href="https://doi.org/10.1101/683722" class="uri">https://doi.org/10.1101/683722</a> (subbmited)</p></li>
<li><p>Husen M. Umer, <strong>Karolina Smolinska-Garbulowska</strong>, Jan Komorowski, Claes Wadelius, Functional annotation of noncoding mutations in cancer, 2019(manuscript)</p></li>
<li><p>Marco Cavalli, Nicholas Baltzer, Gang Pan, José Ramón Bárcenas Walls, <strong>Karolina Smolinska Garbulowska</strong>, Chanchal Kumar, Stanko Skrtic, Jan Komorowski, and Claes Wadelius. Studies of liver tissue identifies functional gene regulatory elements associated to gene expression, type 2 diabetes and other metabolic diseases, Human genomics 13, no. 1 (2019): 20</p></li>
<li><p>Mateusz Garbulowski, Klev Diamanti, <strong>Karolina Smolinska-Garbulowska</strong>, Patricia Stoll, Susanne Bornelöv, Aleksander Øhrn, Jan Komorowski, (2019), R.ROSETTA: a package for analysis of rule-based classification models, bioRxiv 625905; doi: <a href="https://doi.org/10.1101/625905" class="uri">https://doi.org/10.1101/625905</a> These authors contributed equally to the work as second authors.</p>
</h5></li>
</ol>
</section>
<section id="section-1" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2></h2>
<h5>
<p><strong>Posters:</strong></p>
<ol type="1">
<li>Michal J. Dabrowski, Michal Draminski, Klev Diamanti, Mateusz Garbulowski, <strong>Karolina Smolinska</strong>, Jacek Koronacki, Jan Komorowski. Identification of the boundary between common and ancestry-dependent levels of gene expression of Homo sapiens adaptive immune system. Systems Biology: Networks, Cold Spring Harbor Laboratory, Cold Spring Harbor, USA 14-18 March 2017.</li>
<li>Mateusz Garbulowski, <strong>Karolina Smolinska</strong>, Klev Diamanti, Patricia Stoll, Zeeshan Khaliq, Sara Younes, Behrooz Torabi, Susanne Bornelöv, Jan Komorowski. RROSETTA: a toolkit for creating rough set models. Swedish Bioinformatics Workshop 2017, Uppsala, Sweden, 13-14 November 2017.</li>
<li>Mateusz Garbulowski, <strong>Karolina Smolinska</strong>, Klev Diamanti, Patricia Stoll, Zeeshan Khaliq, Sara Younes, Behrooz Torabi, Susanne Bornelöv, Jan Komorowski. RROSETTA: a toolkit for creating rough set models. Swedish Epigenetics and Chromatin Meeting, EpiChrom 2017, Uppsala, Sweden, 30 November - 1 December 2017.</li>
<li><strong>Karolina Smolinska-Garbulowska</strong>, Steven O. O. Anyango, Jan Komorowski, VisuNet: an interactive tool for rule network visualization of rule-based models, SciLifeLab Science Summit 2019, Artificial Intelligence for Life Sciences, Uppsala, May 15, 2019</li>
</ol>
<p><strong>Presentation: </strong></p>
<ol type="1">
<li><strong>Karolina Smolinska-Garbulowska</strong>, VisuNet: an interactive tool for rule network visualization of rule-based models, the awarded Flash Talk, SciLifeLab Science Summit 2019, Artificial Intelligence for Life Sciences, Uppsala, May 15, 2019</li>
</ol>
</h5>
</section>
<section id="acknowledgements" class="slide level2" data-background-image="images/UU-figures-en-2019copy.jpg">
<h2>Acknowledgements</h2>
<div id="left">
<p><img class="plain" src="images/group.png" style="max-width:100%;" align="middle" ></p>
</div>
<div id="right">
<p>Jan Komorowski Laboratory</p>
<p>Claes Wadelius Laboratory</p>
<p>Manfred Grabherr</p>
</div>
</section>
    </div>
  </div>

  <script src="Presentation_ht_9_12_files/reveal.js-3.3.0.1/lib/js/head.min.js"></script>
  <script src="Presentation_ht_9_12_files/reveal.js-3.3.0.1/js/reveal.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://github.com/hakimel/reveal.js#configuration
      Reveal.initialize({
        // Display controls in the bottom right corner
        controls: false,
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,
        // Enable keyboard shortcuts for navigation
        keyboard: true,
        // Vertical centering of slides
        center: false,
        // Enable slide navigation via mouse wheel
        mouseWheel: true,
        // Transition style
        transition: 'slide', // none/fade/slide/convex/concave/zoom
        // Transition style for full page slide backgrounds
        backgroundTransition: 'default', // none/fade/slide/convex/concave/zoom



        // Optional reveal.js plugins
        dependencies: [
        ]
      });
    </script>
  <!-- dynamically load mathjax for compatibility with self-contained -->
  <script>
    (function () {
      var script = document.createElement("script");
      script.type = "text/javascript";
      script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
      document.getElementsByTagName("head")[0].appendChild(script);
    })();
  </script>

<script>
  (function() {
    if (window.jQuery) {
      Reveal.addEventListener( 'slidechanged', function(event) {  
        window.jQuery(event.previousSlide).trigger('hidden');
        window.jQuery(event.currentSlide).trigger('shown');
      });
    }
  })();
</script>


  </body>
</html>
